android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentRemoteFile.java

```diff
@@ -21,7 +21,7 @@ public DocumentRemoteFile(String json) throws JSONException {
             url = json;
         } else {
             JSONObject jsonObject = new JSONObject(json);
-            uuid = jsonObject.optString("uuid");
+            uuid = jsonObject.getString("uuid");
```

Comment:
Do you need this to be mandatory? I think this was the other way because the uuid is not always present in the document data (at least for old forms, that we have to keep compatibility)

Reply:
hmmmm is there any required field for a document in the fold forms?

Reply:
We can test it, but I don't think so, if you use getString it will throw a exception is the value is missing

Reply:
Guys, I'll investigate another approach for this case.